tests/commandnotfound_plugin/commandnotfound.py

```diff
@@ -0,0 +1,14 @@
+from errbot import BotPlugin, cmdfilter
+
+
+class TestCommandNotFoundFilter(BotPlugin):
+    def __init__(self, *args, **kwargs):
+        super(TestCommandNotFoundFilter, self).__init__(*args, **kwargs)
+        self.catch_unprocessed = True
+
+    @cmdfilter
```

Comment:
That makes more sense.
I wasn't a big fan of that __init__/__self__ thing I had done originally...